src/pages/Signup.jsx

```diff
@@ -0,0 +1,45 @@
+import React,{useEffect,useState} from 'react'
+import {Link,useNavigate} from 'react-router-dom'
+import { useAuth } from '../contexts/AuthContext';
+
+export default function Signup() {
+    const [signUpForm,setSignUpForm] = useState({name:'',email:'',password:''})
```

Comment:
try avoid using objects in useState, Using object in useState may cause issues in larger code base, you can either have 3 useStates or useReducer

Reply:
okay, will work on it.